packages/paste-core/components/date-picker/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@twilio-paste/date-picker",
+  "version": "0.0.0",
+  "category": "user input",
+  "status": "alpha",
+  "description": "A form component used to select a date.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "date-fns": "^2.21.3"
+  },
+  "peerDependencies": {
+    "@twilio-paste/anchor": "^5.0.3",
+    "@twilio-paste/box": "^4.0.5",
+    "@twilio-paste/combobox": "^7.0.2",
+    "@twilio-paste/help-text": "^6.0.3",
+    "@twilio-paste/icons": "^5.2.0",
+    "@twilio-paste/input": "^3.0.6",
+    "@twilio-paste/input-box": "^4.0.4",
+    "@twilio-paste/label": "^6.0.3",
+    "@twilio-paste/style-props": "^3.0.1",
+    "@twilio-paste/styling-library": "^0.3.1",
+    "@twilio-paste/text": "^4.0.4",
+    "@twilio-paste/theme": "^5.0.1",
+    "@twilio-paste/types": "^3.1.1",
+    "@twilio-paste/uid-library": "^0.2.1",
```

Comment:
Are we using anchor, box, combobox, help-text, icons, label, text, or uid-library anywhere other than in stories? If not, we can remove those from here and the devDeps.

Reply:
Why don't they need to be dependencies if I'm using them in stories?

Reply:
Storybook has been modified in our repository to resolve (read: find) the src files in each of our packages. So it knows how to find Paste dependencies. This makes it easier to manage and use Storybook. This means we can get hot reloads on our changes to code, and also means that we don't need to couple storybook dependencies to the thing we're shipping in each package.